test/test_autograd.py

```diff
@@ -4837,30 +4841,6 @@ def fn(*inputs):
             setattr(TestAutogradDeviceType, test_name, do_test)
 
 class TestAutogradComplex(TestCase):
-    # remove this test after gradcheck support is added for non-holomorphic functions
-    def test_real(self):
-        x = torch.randn(3, 4, 5, dtype=torch.cdouble, requires_grad=True)
-        x.real.sum().backward()
```

Comment:
Looking at `EXCLUDE_METHOD_FOR_ATTRIBUTES`, you actually skip this test where you access the attribute in the new tests you add. So this won't be tested in the new tests right?

Reply:
as per offline discussion, removed `EXCLUDE_METHOD_FOR_ATTRIBUTES` and `real` and `imag` should now be tested properly.